_posts/2017-05-31-khalid's-initial-thoughts-on-vuejs.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: "Khalid's Initial Thoughts On VueJs"
+title: "Khalid's Initial Thoughts On Vue.js"
 date: 2017-05-31 11:43:15
-tags: [ "vuejs", "JavaScript", "frontend", "clientside" ]
+tags: [ "Vue.js", "JavaScript", "frontend", "clientside" ]
 categories:
-twitter_text: "@buhakmeh initial thoughts on @vuejs"
+twitter_text: ".@buhakmeh initial thoughts on @Vue.js"
 authors: Khalid Abuhakmeh
-image: https://vuejs.org/images/logo.png
-image_credit: VueJs
-image_url: https://vuejs.org
+image: https://Vue.js.org/images/logo.png
```

Comment:
Looks like find/replace got a bit too much. Can you fix all the urls and `@vuejs` in `twitter_text`?